.github/workflows/conformance-e2e.yaml

```diff
@@ -194,6 +194,19 @@ jobs:
             lb-mode: 'snat'
             egress-gateway: 'true'
 
+          - name: '16'
+            # renovate: datasource=docker depName=quay.io/lvh-images/kind
+            kernel: '5.15-20240305.092417'
+            kube-proxy: 'none'
+            kpr: 'strict'
+            devices: '{eth0,eth1}'
+            secondary-network: 'true'
+            tunnel: 'vxlan'
+            lb-mode: 'snat'
+            encryption: 'wireguard'
+            encryption-node: 'false'
+            host-fw: 'true'
+
```

Comment:
@qmonnet this seems to fail reliably in CI for
> north-south-loadbalancing/outside-to-nodeport/curl-0: cilium-test/host-netns-non-cilium-j8jpw (172.18.0.4) -> cilium-test/echo-same-node (echo-same-node.cilium-test:8080)

Let's maybe just drop for now (wireguard on v1.13 isn't very relevant) ?

Reply:
Dropped it